src/app/visual/toolbar/toolbar-visual.component.html

```diff
@@ -247,3 +247,109 @@
     </sky-toolbar-section>
   </sky-toolbar>
 </div>
+
+<div
+  id="screenshot-toolbar-viewkeeper"
+  style="padding-top: 10px;"
+  [skyViewkeeper]="['.sky-toolbar-container']"
+>
+  <sky-toolbar>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-primary"
+      >
+        <!-- The margin style here is to propery space the icon and button text in modern theme. This is not possible in the toolbar item with pure css today because you can't distinguish pure text nodes in css. -->
+        <sky-icon
+          icon="plus-circle"
+          [ngClass]="{
+            'sky-margin-inline-compact' : theme == 'modern'
+          }"
+        >
+        </sky-icon>
+        Button 0
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 1
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 2
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 3
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 4
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 5
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 6
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 7
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-default"
+      >
+        Button 8
+      </button>
+    </sky-toolbar-item>
+    <sky-toolbar-view-actions>
+      <button
+        class="sky-btn sky-btn-default sky-btn-icon"
+        title="Sort descending"
+        type="button"
+      >
+        <sky-icon
+          icon="angle-double-down"
+        >
+        </sky-icon>
+      </button>
+      <button
+        class="sky-btn sky-btn-default sky-btn-icon"
+        title="Sort ascending"
+        type="button"
+      >
+        <sky-icon
+          icon="angle-double-up"
+        >
+        </sky-icon>
+      </button>
+    </sky-toolbar-view-actions>
+  </sky-toolbar>
+  <p *ngFor="let number of [0,1,2,3,4, 5, 6, 7, 8, 9, 10, 11, 12, 13]">
```

Comment:
Looks like the spacing is off inside the array.

Reply:
Fixed